parser-typechecker/src/Unison/Builtin/Terms.hs

```diff
@@ -22,13 +22,15 @@ import Unison.Type qualified as Type
 import Unison.Var (Var)
 import Unison.Var qualified as Var
 
-builtinTermsSrc :: a -> [(Symbol, Term Symbol a, Type Symbol a)]
+builtinTermsSrc :: a -> [(Symbol, a, Term Symbol a, Type Symbol a)]
```

Comment:
Why add this field only to populate it with `()` and throw it away later?

Reply:
It's so that `builtinTermsSrc` matches the correct shape to be combined into a UnisonFile elsewhere.

E.g. see its usage here, the annotation to use is passed in: https://github.com/unisonweb/unison/blob/f51840e6fd60f2e083b1c1059fa069e83f159324/unison-cli/src/Unison/Codebase/Editor/HandleInput.hs#L1312

Reply:
Ok cool. I just wondered because `builtinTermsSrc` didn't show up anywhere else in the diff.